standalone-metastore/metastore-server/pom.xml

```diff
@@ -30,6 +30,12 @@
       <artifactId>hive-standalone-metastore-common</artifactId>
       <version>4.0.0-SNAPSHOT</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-metastore</artifactId>
```

Comment:
Why do you need to add this dependency? I don't think this is required.

Reply:
TestMetastoreConf.java belongs to metastore-server and SerDeStorageSchemaReader belongs to hive-metastore. So i had to add this dependency with test scope. Please guide me if there is any other way around, will make changes accordingly